python-libraries/nanover-omni/src/nanover/omni/omni.py

```diff
@@ -107,17 +107,26 @@ def print_basic_info_and_wait(self):
 
     @property
     def app_server(self):
+        """
+        The NanoVer application server used by this runner.
+        """
         return self._app_server
 
     @property
     def simulation(self):
+        """
+        The currently selected simulation.
+        """
         try:
             return self.simulations[self._simulation_index]
         except IndexError:
             return None
 
     @property
     def is_paused(self):
+        """
+        Is the currently selected simulation paused?
```

Comment:
"""
    Checks if the currently selected simulation is paused.

    Returns:
        bool: True if the simulation is paused, False otherwise. Returns None if no runner is available.
    """